app/views/models/welcome.html.erb

```diff
@@ -0,0 +1,11 @@
+<% content_for(:title, _("Hardware Models")) %>
+<div class="blank-slate-pf">
+  <div class="blank-slate-pf-icon">
+    <%= icon_text("info-circle", "", :kind => "fa") %>
```

Comment:
This should perhaps be `laptop` or `server`, something recognisable as a type of computer.